### ISSUE TITLE
Find better ways to deal with issues related to ambiguous variable names in `consolidate()` 

### ISSUE THREAD

**Issue by henriquesposito** [globalgov/manydata#201]
The function does not run properly when variable prefixes for different variables are the same (it assumes these are variables to be "consolidated"). Now we can deal with the issue several different ways, some (not mutually exclusive) options are below:

- Add an item on the PR template for many packages about making sure developers check if their databases work with consolidate before a version release
- Add more documentation/instructions for developers about variable naming practices expected for many packages
- Make the warnings in consolidate() more informative
- Make consolidate interactive (ask users) for when variable prefixes match but affixes are not the same (or the expected ones)

**Comment by jhollway** [comment]
Can you give an example?

**Comment by henriquesposito** [comment]
In manyenviron, for example, in the agreements database we have variables named "Lineage" and "L" across different datasets. Since these variables start with the same prefix and we use "dplyr::starts_with()" to identify and consolidate variables into from different datasets, consolidate() thinks these are the same variable. Therefore, consolidate does not work with some of the databases in manyenviron. 

This is why we opened a PR on manyenviron and on manytrade, so that once manydata is on CRAN users can actually successfully "consolidate" the databases in our packages. As for manystates, consolidate() works with the released branch (main) but not with the develop branch. I opened an issue there so that we can take fix these variable name inconsistencies before releasing a new version. We are also checking for this in the other data packages.

I have tried to play around a bit and change how consolidate() works but I did not find a way to change the function. Though I think we can add more informative warnings for users about when variable names are ambiguous to consolidate() and make sure data contributors in the many universe are aware of these requirements.
